saito-core/src/core/data/configuration.rs

```diff
@@ -31,10 +31,12 @@ pub struct Server {
     pub stat_timer_in_ms: u64,
     pub thread_sleep_time_in_ms: u64,
     pub block_fetch_batch_size: u64,
+    pub reconnection_wait_time:u32
```

Comment:
name doesn't match with the field in json.

Also what should be edited for git is config.template.json